flow/project.py

```diff
@@ -4271,6 +4288,14 @@ def _register_groups(self):
                 operation_name
             ] = directives
 
+    def reregister_aggregates(self):
```

Comment:
Is it possible to combine this logic with the original registration, so that there is only one implementation used by "registration" and "re-registration"?

Reply:
Can you remind me why exactly this function is necessary to make part of the API again? Is it because the user needs to tell flow when jobs have been added to the data space? The implementation is different enough from the way that aggregators are determined inside `_register_groups` to give me a little pause.

Reply:
I agree this is worth further investigation. For now, we made the method private in e52aa43. I am going to document this in a separate TODO issue.